tuify/src/components/select_component.rs

```diff
@@ -114,7 +114,7 @@ impl<W: Write> FunctionComponent<W, State> for SelectComponent<W> {
                 let padding_left = " ".repeat(start_display_col_offset);
                 let index = caret_row_scroll_adj + 1;
                 if is_selected {
-                    format!("{padding_left} > {index} ")
+                    format!("{padding_left} ►▶ {index} ")
```

Comment:
# Option 1
Instead of 2 characters, I think it looks better w/ a single character. Here are some examples:
- pointer: "❯",

# Option 2
Another idea is to use a set of the following symbols
- radioOn: "◉",
- radioOff: "◯",

So if something is selected it might get `radioOn` & if it is unselected it might get `radioOff`.

# Question
Do you think you can implement these changes?

Reply:
@nisargparikh69 That looks great! 👍🏽